### DISCUSSION TITLE
HRV status data

### DISCUSSION THREAD

**Discussion by sloth201** [tcgoetz/GarminDB#186] [General]
I am trying to find the HRV status parameter. The HRV_STATUS fit files are downloaded but I can't see this in the SQlite database - do I need to do something else to import this data into the database?

**Comment by aracg** [comment]
were you able to figure this one out?

**Comment by ekr1** [comment]
Seconding this question to receive notifications when there is an update. The relevant source https://github.com/tcgoetz/GarminDB/blob/master/garmindb/monitoring_fit_file_processor.py does not contain code for parsing/writing the HRV_STATUS fitfiles. I would love to have this data, but don't have time to try and implement it right now. Let's see if someone picks up the torch. ;)

**Comment by joelgerard** [comment]
I had access to Gemini Pro (FWIW), and I added HRV [here](https://github.com/tcgoetz/GarminDB/compare/master...joelgerard:GarminDB:master). Not sure if you want an AI PR, or if you want to merge it yourself, but thought I'd point it out. Love this repos, btw.

**Comment by ekr1** [comment]
Nice, @joelgerard. Looking forward to this being merged. Looking at your code it seems like when it is, no further action on the user's side is required (provided they already run garmindb with the `--all` option)?

**Reply by joelgerard** [reply]
I squashed the commits and posted a PR. I excluded all the build changes I had to do to get it building on my machine. Pretty sure I got it right...

**Comment by hellotheremike** [comment]
Any updates on this @ekr1 & @joelgerard?

**Reply by ekr1** [reply]
I'm just a bystander here ;). @joelgerard 's PR seems to have been closed without merge: https://github.com/tcgoetz/GarminDB/pull/298

**Reply by nowszymon** [reply]
@joelgerard any chance of merging develop to master anytime soon?

**Reply by tcgoetz** [reply]
> any chance of merging develop to master anytime soon?

https://github.com/tcgoetz/GarminDB/releases/tag/v3.7.0

**Comment by tcgoetz** [comment]
develop branch has new fit processing including HRV
